layouts/index.html

```diff
@@ -28,9 +28,8 @@ <h2 class="subtitle is-5 is-muted">
                 </p>  */}}
             </div>
             <div class="column is-5 is-offset-1">
-                <figure class="image"> 
-                {{- $img := resources.Get $image -}}
-                <img src="{{- $img.Permalink | relURL -}}" 
+                <figure class="image">
+                <img src="{{- .Site.Params.homepage.image -}}" 
```

Comment:
The reason why I was not doing this it is because you are linking a static image, i.e.,  the resize will be done at the browser level, therefore the original image (potentially big) will be transferred over the internet, not ideal for poor connections. 
I need to resize using something like:
```
{{- $img := resources.Get $imagePath -}}
{{- $image := img.Resize $size -}}
```
In this way, when you generate the site, Hugo engine will create a new image with the specified dimensions.
In this page I am actually resizing at browser level using the css style but I don't know if I forgot this step or if it was intentional (anyway I need to add it).

Reply:
I see. Let me know if you want me to open a PR with only the footer renaming? Cheers!

Reply:
Yes please! If you can create a PR only for the footer will be amazing. I will add also an option to add a title to the link instead of the default.